src/formatter.ts

```diff
@@ -97,15 +97,19 @@ export default class Formatter implements vsc.DocumentFormattingEditProvider,
     }
 };
 
-const languageMap = {
-    'apex': 'JAVA',
-    'c': 'C',
-    'cpp': 'CPP',
-    'csharp': 'CS',
-    'd': 'D',
-    'java': 'JAVA',
-    'objective-c': 'OC',
-    'pawn': 'PAWN',
-    'pde': 'JAVA',
-    'vala': 'VALA'
-};
+const langOverrides = vsc.workspace.getConfiguration("uncrustify").get("langOverrides", false);
+
+const languageMap = Object.assign(
+  {
+    c: "C",
+    cpp: "CPP",
+    csharp: "CS",
+    d: "D",
+    java: "JAVA",
+    "objective-c": "OC",
+    pawn: "PAWN",
+    pde: "JAVA",
+    vala: "VALA"
+  },
+  langOverrides
+);
```

Comment:
Now this is just me being picky, but the rest of the code is using 4 spaces indentation instead of just 2 like here. I simply like projects to be consistent about formatting :)